geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionClearDUnitTest.java

```diff
@@ -430,13 +423,54 @@ public void normalClearFromClient() {
         .isEqualTo(0);
   }
 
+  @Test
+  public void testClearTime() {
+    configureServers(false, true);
+    client1.invoke(this::initClientCache);
+    client2.invoke(this::initClientCache);
+
+    accessor.invoke(() -> feed(false));
+    verifyServerRegionSize(NUM_ENTRIES);
+    dataStore1.invoke(() -> {
+      PartitionedRegion partitionedRegion = (PartitionedRegion) getRegion(false);
+      assertThat(partitionedRegion.getCachePerfStats().getClearTime()).isEqualTo(0L);
+    });
+    dataStore1.invoke(() -> getRegion(false).clear());
+    verifyServerRegionSize(0);
+    dataStore1.invoke(() -> {
+      PartitionedRegion partitionedRegion = (PartitionedRegion) getRegion(false);
+      assertThat(partitionedRegion.getCachePerfStats().getClearTime()).isGreaterThan(0L);
```

Comment:
It's better to use "<10" instead of ">0". It maybe 0 in some fast machine.

Reply:
Hmm. Less than 10 will always be true. This test is basically unnecessary if a fast machine can result in 0 nanoseconds. I find it hard to believe though that a machine can do a clear in 0 nanoseconds. Is that really the case?

Reply:
It depends on the resolution of the system clock. If you keep calling nanoTime you will see it does not always change even though time has elapsed. So it is possible that for a quick clear start and end will get the same nanoTime and elapsed will be zero. It is also possible that it will be more than 10 due to a gc pause. This type of test will tend to always be flaky. If you can modify geode to use a clock provided by the test (I think some support for this was added for unit tests) then you could have a clock that always changes by 1 each time it is called and then you could assert it is > 0 safely. But it might be hard to even then assert more because the clock might be used by other parts of geode updating other stats.

Reply:
can a StatisticsClock be added by the test to this test's cache? Otherwise it could fail intermittently. If not I think it would be better for this test to just test the clear count stat and not the clear time stat. The clear time can be tested in an integration test. I also don't like the idea of doing a bunch of work just to get a stat to change.

Reply:
I am testing this in a JUnit test and so I am just going to pull the Dunit test.